### ISSUE TITLE
Opening SDialogFragment no effect

### ISSUE THREAD

**Issue by ru-nekit-android** [Prototik/HoloEverywhere#126]
I try to open SDialogFragment

public class MyFragment extends SFragment
{

NotePadDialog notePadDialog;

```
private void showNotePadEditor() {
    FragmentManager fm = getActivity().getSupportFragmentManager();
    notePadDialog = NotePadDialog.getInstance(this);
    notePadDialog.show(fm, "fragment_notepad");
}
```

}

public class NotePadDialog extends SDialogFragment{
public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState)
    {
        View view = inflater.inflate(R.layout.fragment_note_pad, container, false);
        return view;
    }
}

But no effect:(

**Comment by ru-nekit-android** [comment]
I use DialogFragment - all fine

**Comment by Prototik** [comment]
Fixed in 1.3
